core/templates/pages/contributor-dashboard-page/services/translate-text.service.spec.ts

```diff
@@ -151,4 +152,28 @@ describe('TranslateTextService', () => {
         expect(textAndAvailability).toEqual(textAndPreviousAvailability);
       }));
   });
+
+  describe('getMachineTranslationAsync', () => {
+    it('should not retrieve translation until dropdown chevron is clicked',
```

Comment:
This is a service with no UI. What has it got to do with the "dropdown chevron"?